packages/utils/smash.js

```diff
@@ -5,17 +5,17 @@
  * @returns { p: { a: 1, b: 2, c: { d: 1, e: 2 } } }
  */
 function main(args) {
-    let parent = args['$fieldName']
+    let parent = args['$fieldName'];
```

Comment:
>then it should be done consistently.

Make "consistency" great again.

Something similar exist for other languages in our repos? Java, Scala, Python, Swift? that will help us with contributions in terms of style.